### ISSUE TITLE
Add aria labels candidate list and global search

### ISSUE THREAD

**Issue by Elblinator** [OpenSlides/openslides-client#6283]
**Describe the bug**
same as https://github.com/OpenSlides/openslides-client/issues/6140

The add me and remove me labels are missing in the assignment list

 and the global search label is not being translated

**Expected behavior**

The missing labels should be added and the not translated label should be translated
